builtin/common/vector.lua

```diff
@@ -1,6 +1,18 @@
 
 vector = {}
 
+function vector.from(x, y, z)
+	return {x = x, y = y, z = z}
+end
+
+function vector.zero()
+	return {x = 0, y = 0, z = 0}
+end
+
+function vector.copy(v)
+	return {x = v.x, y = v.y, z = v.z}
```

Comment:
Should check `v.x`, `v.y` and `v.z` are not nil (like in former new() function)

Reply:
Done.